### DISCUSSION TITLE
[GŁOSOWANIE] 42 spotkanie grupy

### DISCUSSION THREAD

**Discussion by eklime** [AzureCommunityPL/group#17] [Pomoc] [answered]
## Hej!

Już jest - pierwsze, oficjalne głosowanie na sesje na najliższe, wrześniowe spotkanie naszej grupy.

Link do głosowania o tutaj: https://azurecommunitypl.typeform.com/to/RERmbGwg

**Comment by wi5nia** [comment]
Zapowiada sie zacnie 😃

**Reply by huczas** [reply]
da się wielokrotnie głosować z jednego kompa...

**Reply by eklime** [reply]
@huczas wierzymy w uczciwość głosujących ;P

**Comment by kaluzaaa** [comment, marked as answer]
Mamy wyniki! Poniżej wybrane sesje i wyniki głosowania. Jako że Kamil zgłosił dwie, to została wybrana ta z większą ilością głosów 😀

Kolejność | Imię i nazwisko | Tytuł sesji | Długość sesji | Poziom sesji
-- | -- | -- | -- | --
1 | Agnieszka Cent | Partycje w kolejkach | 30 | 200
2 | Hubert Witkowski | Automatyczne budowanie obrazu dla Azure WVD | 45 | 200
3 | Piotr Wolszakiewicz | Godzilla - case study of system collecting data | 45 | 300
4 | Kamil Mrzygłód | ARM on steroids czyli Azure Deployment Manager | 45 | 400

![image](https://user-images.githubusercontent.com/4700951/91039419-d1ad1a00-e60c-11ea-84a4-8a0e541e4c70.png)
